controller/controller.go

```diff
@@ -56,6 +56,11 @@ var (
 	// may adjust this process-wide default.  For finer control, invoke
 	// Run on the controller directly.
 	DefaultThreadsPerController = 2
+
+	// alwaysTrue is the default FilterFunc, which allows all objects
+	// passed through this is used in the default case for passing all
+	// objects to the slowlane from the passed sharedInformer
```

Comment:
super nit: I think there's a bit of missing punctuation here -
```suggestion
	// alwaysTrue is the default FilterFunc, which allows all objects
	// passed through. This is used in the default case for passing all
	// objects to the slowlane from the passed sharedInformer.
```